Hobbify-Stats/model.py

```diff
@@ -0,0 +1,46 @@
+import querys
+
+
+
+
+def relative_freq(username):
+    '''
+    This function returns a number between 0 and 1 that is named relative frequency. After,  realize a conversion to numbers between 0 and 4 for use in the  heatmap development in ReactJS 
+    '''
+
+    total= querys.countIntancesPending(username)[0]
+    done = querys.countIntancesDone(username)[0]
+
+    if total == 0:
+        return f'No hay registros'
+    else:
+        score = done / total  
+
+        if score == 0:
+            return 0
+        elif score > 0 and score <= 0.25:
+            return 1
+        elif score > 0.25 and score <= 0.5:
+            return 2
+        elif score > 0.5 and score <= 0.75:
+            return 3
+        else:
+            return 4
```

Comment:
If you decide to add or remove variables, maybe a continuous-to-discrete mapping can save you some code. It is really just an unnecessary optimization, but nice anyway.